### PR TITLE
include and use queue

### PR DESCRIPTION
**Pull request by mikowitz**
Include the node module `queue` and use it to process an unknown number of records passed to the lambda function.

### CODE REVIEW

File: lambda/index.js [hunk 1]
```diff
@@ -1,21 +1,40 @@
...
+const queue = require('queue');
+
+function processRecord(record) {
+  const child = exec('./ruby_wrapper ' + "'" +  JSON.stringify(record) + "' '" + JSON.stringify(context) + "'", (result) => {
+    context.done(result);
+  });
+
+  child.stdout.on('data', function (data) {
+    data.split("\n").forEach(function(x) {
+      console.log(x);
```
Comment (bjackson2) [inline lambda/index.js:11]: What is the benefit of logging to the console here?

**Comment by bjackson2** [comment]
Just checking about console logging...  otherwise LGTM.

File: lambda/index.js (hunk 1 above)
Comment (mikowitz) [reply lambda/index.js:11]: This logs to AWS CloudWatch.

Comment (mikowitz) [reply lambda/index.js:11]: Which, to give a bit more context, is how we get logs/stats into DataDog and SumoLogic.

**Review by jaeger401 — APPROVED** [review]
Good find. LGTM!

**Pull request merged**